### PR TITLE
Add user microposts

### PR DESCRIPTION
**Pull request by LeCaoDat**
image

### CODE REVIEW

**Comment by LeCaoDat** [comment]
ready

File: app/assets/javascripts/custom.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+$(document).ready(function () {
+  $("#micropost_picture").bind("change", function() {
```
Comment (tranducquoc) [inline app/assets/javascripts/custom.js]: code js dùng nháy ' '

**Comment by tranducquoc** [comment]
commented

**Comment by LeCaoDat** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**